docs/tutorials/buttons.rst

```diff
@@ -159,31 +168,35 @@ pleasant game, so perhaps you can figure out how to improve it.)::
 
     display.clear()
 
-Can you see how we check what buttons are pressed? We used ``if``,
-``elif`` (short for "else if") and ``else``. These are called *conditionals*
-and work like this::
+Czy widzisz jak sprawdzamy jakie przyciski są wciśnięte? Użyliśmy ``if`` (ang. jeśli),
+``elif`` (skrót od "else if") (ang. jeśli jednak) oraz ``else`` (ang. w pozostałych przypadkach).
+Są one nazywane *warunkami* i
+działają tak::
 
-    if something is True:
-        # do one thing
-    elif some other thing is True:
-        # do another thing
+    if coś jest True:
+        # zrób pierwszą rzecz
+    elif coś innego jest True:
+        # zrób następną rzecz
     else:
-        # do yet another thing.
-
-This is remarkably similar to English!
-
-The ``is_pressed`` method only produces two results: ``True`` or ``False``.
-If you're pressing the button it returns ``True``, otherwise it returns
-``False``. The code above is saying, in English, "for ever and ever, if
-button A is pressed then show a happy face, else if button B is pressed break
-out of the loop, otherwise display a sad face." We break out of the loop (stop
-the program running for ever and ever) with the ``break`` statement.
-
-At the very end, when the cyber-pet is dead, we ``clear`` the display.
-
-Can you think of ways to make this game less tragic? How would you check if
-*both* buttons are pressed? (Hint: Python has ``and``, ``or`` and ``not``
-logical operators to help check multiple truth statements (things that
-produce either ``True`` or ``False`` results).
+        # zrób jeszcze coś.
+
+To jest podobne do języka angielskiego!
+
+Metoda ``is_pressed`` generuje jeden z dwóch wyników: ``True`` albo ``False``.
+Jeżeli naciśniesz przycisk, to zwróci ``True``, w przeciwnym wypadku zwróci
+``False``. Powyższy kod można przetłumaczyć jako "jeżeli i tylko wtedy gdy
+przycisk A został naciśnięty, to pokaż szczęśliwą twarz, jeżeli przycisk B
+został naciśnięty, to przerwij pętlę, a w przeciwnym wypadku pokaż smutną
+minę." Przerywamy wykonywanie pętli (zatrzymujemy uruchomiony w nieskończoność
+program) za pomocą instrukcji ``break``.
+
+Na samym końcu, kiedy cyber-zwierzak nie żyje, czyścimy ekran metodą
+``clear``.
+
+Czy możesz pomyśleć o sposobie, aby ta gra była mniej tragiczna? Jak chciałbyś
+sprawdzić, czy *oba* przyciski zostały naciśnięte? (Podpowiedź: Python ma
+logiczne operatory  ``and``, ``or`` i ``not``, które umożliwiają sprawdzenie
```

Comment:
operatory logiczne

Reply:
może warto dać tu tłumaczenia tych słów, a przykład:

Python ma operatory logiczne "i" (ang. ``and``), "lub" (ang. ``or``) oraz "nie" (ang. ``not``), którymi można sprawdzić wiele warunków (wyrażeń, które mają wartość ``True`` lub ``False``).